wideq/client.py

```diff
@@ -138,6 +139,24 @@ def get_device(self, device_id) -> Optional['DeviceInfo']:
                 return device
         return None
 
+    def get_device_class(self, device_id):
+        """Look up a subclass of Device object by device ID.
+
+        Return a Device instance if no subclass exists for the device type.
+        Return None if the device does not exist.
+        """
+        from . import util
+
+        deviceInfo = self.get_device(device_id)
+        if not deviceInfo:
+            return None
+        classes = util.device_classes()
+        if deviceInfo.type in classes:
+            return classes.get(deviceInfo.type)(self, deviceInfo)
```

Comment:
```suggestion
            return classes[deviceInfo.type](self, deviceInfo)
```

…because we already know the class is in the map.

Reply:
I don't understand the difference but I trust you about this pattern